src/client/transport/local.js

```diff
@@ -208,13 +210,13 @@ const localMasters = new Map();
 export function Local(opts) {
   return transportOpts => {
     let master;
-
-    if (localMasters.has(transportOpts.gameKey) & !opts) {
+    if (localMasters.has(transportOpts.gameKey)) {
       master = localMasters.get(transportOpts.gameKey);
     } else {
       master = new LocalMaster({
         game: transportOpts.game,
         bots: opts && opts.bots,
+        persist: opts && opts.persist,
```

Comment:
Maybe allow passing the localstorage key to be used here? That way if the user wants to "wipe" the data they can just switch the key to a new value.

Reply:
Hmm, will think probably you are right it could be useful.